conf/awsbatch.config

```
@@ -1,6 +1,6 @@
 /*
  * -------------------------------------------------
- *  Nextflow config file for AWS Batch
+ *  Nextflow config file for AWS Batch [not used for lncpipe]
```

```suggestion
 *  Nextflow config file for AWS Batch
```

Someone might want to use it there, just keep it in ;-)

---

Okey